algoliasearch-common/src/main/java/com/algolia/search/AsyncAPIClient.java

```diff
@@ -1036,14 +1037,19 @@ public String generateSecuredApiKey(
 
   CompletableFuture<AsyncTaskSingleIndex> partialUpdateObjects(
       String indexName, List<Object> objects, RequestOptions requestOptions) {
+    return partialUpdateObjects(indexName, objects, requestOptions, false);
```

Comment:
Shouldn't it be `true` by default? Else we introduce a breaking change in the default behavior of `partialUpdateObjects` (used to create by default, now would _not_ create by default) 🤔